pandas/core/strings.py

```diff
@@ -51,11 +51,19 @@ def str_cat(arr, others=None, sep=None, na_rep=None):
     """
     Concatenate strings in the Series/Index with given separator.
 
+    This function concatenates elements of Series/Index and elements
+    of lists or list-like objects having same length.
+    The concatenation is done at corresponding indices of
+    iterable specified by `Series` when two or more iterables
```

Comment:
This part is not fully clear to me: " corresponding indices of iterable"

Can you give a small example of what you want to explain here?

Reply:
Corresponding indices indicates that the concatenation is being done in one to one correspondence with the indices i.e, index of iterable, let say list.

For example :
>>> pd.Series(['a', 'b', 'c']).str.cat(['A', 'B', 'C'], sep=',')
    index
    0             a,A (a with A i.e, 0th index of list in Series and 0th index of list in cat(list...) ) 
    1             b,B (b with B)
    2             c,C (c with C)

Reply:
Ah, OK. In that case, I think it is fine to say that "concatenates the Series/Index and `other` element-wise"